packages/manager/modules/netapp/src/dashboard/volumes/dashboard/template.html

```diff
@@ -0,0 +1,274 @@
+<div>
+    <oui-header
+        heading="{{:: $ctrl.volume.name }}"
+        description="{{:: $ctrl.volume.description }}"
+    >
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                href="{{:: $ctrl.volumeDashboardLink }}"
+                active="$ctrl.currentActiveLink() === $ctrl.volumeDashboardLink"
+            >
+                <span
+                    data-translate="netapp_volumes_dashboard_general_information"
+                ></span>
+            </oui-header-tabs-item>
+        </oui-header-tabs>
+    </oui-header>
+    <div data-ovh-alert></div>
+    <div ui-view>
+        <div class="row">
+            <oui-tile
+                class="col-md-3 mr-2"
+                heading="{{:: 'netapp_volumes_dashboard_information' | translate}}"
+            >
+                <oui-tile-definition
+                    term="{{:: 'netapp_volumes_dashboard_information_name' | translate }}"
+                >
+                    <oui-tile-description data-ng-if="!$ctrl.isEditing.name">
```

Comment:
Probably, toggling the edit name will be required in many places going forward. This can be moved to manager-components if possible :)